site/profile/manifests/efa.pp

```diff
@@ -0,0 +1,35 @@
+class profile::efa (
+  String $version = 'latest'
+) {
+  package { 'libibverbs-utils':
+    ensure => 'installed',
+  }
+
+  package { 'rdma-core-devel':
+    ensure => 'installed',
+  }
+
+  package { 'librdmacm-utils':
+    ensure => 'installed',
+  }
+
+  archive { 'download-efa-driver':
+    path         => "/opt/puppetlabs/puppet/cache/puppet-archive/aws-efa-installer-${version}.tar.gz",
+    extract      => true,
+    extract_path => '/tmp/',
+    source       => "https://efa-installer.amazonaws.com/aws-efa-installer-${version}.tar.gz"
+  }
+
+  exec { 'install-efa-driver':
+    command => 'bash efa_installer.sh -y && rm -rf /tmp/aws-efa-installer',
```

Comment:
In cloud-init, we used the flag `--minimal`, is it no longer recommended to use it?

https://github.com/ComputeCanada/magic_castle/blob/main/common/configuration/puppet.yaml#L116